include/boost/stacktrace/detail/frame_msvc.ipp

```diff
@@ -28,9 +28,11 @@
 
 
 #ifdef __CRT_UUID_DECL // for __MINGW32__
+#if !defined(__MINGW32__) || (__GNUC__ < 12) // https://github.com/boostorg/stacktrace/issues/133
```

Comment:
Thanks for sending a fix!

Can we accommodate mingw-llvm as well? I think we'd have to use something like this:

```
#if !defined(__MINGW32__) || \
     ((!defined(__clang__) && __GNUC__ < 12) || \
      (defined(__clang__) && __clang_major__ < 16))
```

Here are the macros defined by mingw-llvm 16 (https://github.com/mstorsjo/llvm-mingw):

```
#define __GNUC_GNU_INLINE__ 1
#define __GNUC_MINOR__ 2
#define __GNUC_PATCHLEVEL__ 1
#define __GNUC__ 4

#define __clang_major__ 16
#define __clang_minor__ 0
#define __clang_patchlevel__ 0
```

Reply:
Done.